subsys/bluetooth/common/Kconfig

```
@@ -14,13 +14,17 @@ config BT_BUF_ACL_TX_SIZE
 	help
 	  Maximum supported ACL size of data packets sent from the Host to the
 	  Controller. This value does not include the HCI ACL header.
-	  The Host will segment the data transmitted to the Controller so that
+
+	  The Host will fragment the data transmitted to the Controller so that
```

```suggestion
	  The Host will segment the data transmitted to the Controller so that
```

`BT_BUF_ACL_TX_SIZE` is the maximum ACL data size, not the maximum HCI ACL data fragment size. The fragmentation size may be bigger or smaller than this.

TBH, this sentence is super vague. I'm not sure about the value of keeping it.

---

yeah, let's just remove it. we were thinking of adding the same kind of docs for host - controller data flow after this PR anyways.